roles/internal/webserver-app/tasks/solr.yml

```diff
@@ -3,6 +3,11 @@
 
 # Only tasks related to configuring search_api_solr are put here
 # Solr server configuration should go into the solr playbook: https://github.com/Islandora-Devops/claw-playbook/blob/dev/solr.yml
+- name: Add a patch for symfony-event-dispatcher per the INSTALL.md docs on search_api_solr:3.8
+  composer:
+    command: require
+    arguments: symfony/event-dispatcher:'4.3.4 as 3.4.99' drupal/search_api_solr
```

Comment:
Replace with: `arguments: symfony/event-dispatcher:'4.3.4 as 3.4.99' solarium/solarium:^5.1.4 drupal/search_api_solr`

Reply:
ok, i'll update this

Reply:
Actually, there is a 5.1.6 already, so maybe we should use `^5.1`. I think composer should run with that...

Reply:
ok, spinning that up now